examples/complex_module/run.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+cd `dirname $0`
+
+# Create a virtual environment to run our code
+VENV_NAME="venv"
```

Comment:
Does this not assume you have `venv` installed?  and a virtual env named `venv`?

Reply:
venv is baked into python 3.4 and up, so should be good for the versions of python we support.
I've used `venv` for myself for the longest time haha, is there a different name you like?

Reply:
AH sorry this is creating a virtual env. I guess then it assumes that there isn't already a virtual env named `venv`? What would happen if a user already have a `venv`?

Reply:
NVM, it creates it in this directory, so there should be no collisions.